libs/database/src/lib/entities/localization-keys.entity.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  AllowNull,
+  AutoIncrement,
+  Column,
+  Model, NotNull,
+  PrimaryKey,
+  Table,
+  Unique
+} from "sequelize-typescript";
+import { BIGINT, INTEGER, NUMBER, STRING, TEXT } from "sequelize";
+
+@Table({ tableName: "localization_keys", underscored: true })
+export class LocalizationKey extends Model<LocalizationKey> {
+
+  @PrimaryKey
+  @AutoIncrement
+  @Column(BIGINT.UNSIGNED)
+  override id: number;
+
+  @NotNull
+  @AllowNull(false)
```

Comment:
Why are both of these on here? I'm pretty sure not null is the default, so these may both be removed. Also applies to `value` below. Finally, if a column is not nullable, the TS type of the field should not include `null`. So this one should be:
```
@Column(STRING)
key: string;
```